Formula/semtag.rb

```diff
@@ -0,0 +1,14 @@
+class Semtag < Formula
+  desc "Semantic tagging script for git"
+  homepage "https://github.com/pnikosis/semtag"
+  url "https://github.com/pnikosis/semtag/archive/88ece2b24c89d94619e41480c224710ea197bee7.tar.gz"
+  sha256 "d64badcdad5d32c813d8de8a9bd6cd2a3826abfc446e796866ed30cc6199f90e"
```

Comment:
```suggestion
  url "https://github.com/pnikosis/semtag/archive/v0.1.0.tar.gz"
  sha256 "9e16a418b795363a9283318e5dc02da8a67ec96adaac0ae8eb23f1b21b06666f"
```

We accepted tagged releases only.